tasks/cores-post93.yml

```
@@ -0,0 +1,32 @@
+---
+- name: Check current list of Solr cores.
+  uri:
+    url: http://{{ solr_jetty_host }}:{{ solr_port }}/solr/admin/cores
```

On pourrait améliorer ça en faisant quelque chose dans le genre:
```suggestion
    url: http://{{ solr_jetty_host if solr_jetty_host != '0.0.0.0' else solr_admin_listen_ip }}:{{ solr_port }}/solr/admin/cores
```
Il faut mettre une variable solr_admin_listen_ip configurée à 127.0.0.1 par défaut, comme ça on laisse le choix de pouvoir la surcharger.